### PR TITLE
BaSP-TG-53: CHANGE GetAll method returns 200 when list is empty

### PR DESCRIPTION
**Pull request by fede230298**
CHANGE all entities to send status 200 when fetching an empty list.
RE-ARRANGE test to validate the new status and messages

### CODE REVIEW

File: src/controllers/admins.js [hunk 1]
```diff
@@ -65,8 +65,9 @@ const getAllAdmins = async (req, res) => {
   try {
     const admins = await Admins.find(req.query);
     if (!admins.length) {
-      return res.status(404).json({
-        message: 'Admin not found',
+      return res.status(200).json({
```
Comment (Ignacio-Sormani) [inline src/controllers/admins.js:68]: Both success responses are the same. I think we should just keep one return 200 with same message: 'Admin List"

Comment (Ignacio-Sormani) [reply src/controllers/admins.js:68]: Apply same comment for all the controllers

**Review by Ignacio-Sormani — CHANGES_REQUESTED** [review]
I see that you only updated tests for 3 resources. Are there missing tests? Please add them

File: src/controllers/admins.js (hunk 1 above)
Comment (fede230298) [reply src/controllers/admins.js:68]: So I should remove the conditional !entity.length and leave just the 200 status when OK?

Comment (Ignacio-Sormani) [reply src/controllers/admins.js:68]: Yes, it will always return the array even if it's empty

**Review by GonzaloAsencio — APPROVED** [review]

**Review by Ignacio-Sormani — APPROVED** [review]

**Review by javierperezraffo — APPROVED** [review]

**Pull request merged**